wagtail/tests/dummy_sendfile_backend.py

```diff
@@ -0,0 +1,8 @@
+from django.http import HttpResponse
```

Comment:
Yep, it's a convention. If we forget to add it somewhere that it's needed it could cause issues for Python 2 users. To prevent that from happening we've configured isort (https://github.com/torchbox/wagtail/blob/master/setup.cfg#L8) to make sure it's in every single file.